src/tito/release/main.py

```diff
@@ -527,6 +527,7 @@ def _koji_release(self):
                 if scl:
                     builder = copy.copy(self.builder)
                     builder.scl = scl
+                    builder.cleanup()
```

Comment:
Sorta ad-hoc testing changes to start with, this worked for me originally for scl and non-scl, but @jlsherril is reporting failures after adding the package to say a whitelist in the first non-scl releaser.